src/simulations/optimisation.py

```diff
@@ -638,8 +638,11 @@ def _problem(self):
 
         # save results
         res = self._save_results(p.variables)
+        number_opti_constraints = len(p.constraints)
+        if 'n_opti_constraints' not in prm['syst']:
+            prm['syst']['n_opti_constraints'] = number_opti_constraints
```

Comment:
you can access the object attribute as

`self.syst['n_opti_constraints'] = number_opti_constraints`

Reply:
That makes a lot of sense, thanks.